processor/casttosumprocessor/config.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package casttosumprocessor
+
+import (
+	"fmt"
+
+	"go.opentelemetry.io/collector/config"
+)
+
+// Config defines the configuration for the processor.
+type Config struct {
+	*config.ProcessorSettings `mapstructure:"-"`
+
+	// List of input metrics to convert to sums
+	Metrics []string `mapstructure:"metrics"`
```

Comment:
Name it `cast_to_sum_metrics` so we can later add other forms of casting if they come up?

Reply:
We could also create multiple instances of this processor with different options, right? Then each of them would have the `metrics` parameter…

I should probably change the above comment to just `// List of input metrics`.